test/runtests.jl

```diff
@@ -110,4 +110,15 @@ Threads.nthreads() <= 1 && @info "Threads.nthreads() = $(Threads.nthreads()), mu
             end
         end
     end
+
+    @testset "EXR" begin
+        for typ in [RGBA{Float16}]
```

Comment:
Just a question, is this OpenEXR reserved for `RGBA{Float16}` type? One thing that might be worth noting is that `Float16` arithmetic is implemented in software so doing computation on it can be slow.

Maybe also add some other type, e.g., `Gray{N0f8}`, `RGB{Float32}` to make sure this backend does support generic types?

Reply:
The underlying C API of the openexr library only supports saving/loading values isomorphic to `RGBA{Float16}`. Any other types would need to be converted to/from that. The C++ interface is a bit more flexible, but wrapping it for julia is well beyond the scope of what I'm willing to do myself at this point.

More specifically, the loader can read EXR files storing other numeric types, but they will be converted to 16-bit floats in memory. The saver will only write 16-bit floats, but flags can be thrown to save to disk only a subset of the 4 channels in the given RGBA image.

I can add more dummy files to `test/files` that store subsets of channels and/or samples at different bit depths, and show that the loading still succeeds (though always returning an `Array{RGBA{Float16},2}`). Would that address what you are asking about here?

OpenEXR.jl could use some tests for certain.

Reply:
Since only `RGBA{Float16}` type is supported, we can leave the test as it is for now. Other read/write tests can go to OpenEXR's own package.

Reply:
Testset coverage extended to basically all color types.